### DISCUSSION TITLE
How to deploy nextjs app router app on cpanel

### DISCUSSION THREAD

**Discussion by Uri3lArchangel** [vercel/next.js#65600] [App Router]
Hi, I am currently trying to deploy my nextjs app (app router) on cpanel, it has some dynamic routes also.

**Comment by ghost** [comment]
Maybe 🥶 https://tech-scan.com/blog/how-to-host-nextjs-14-web-app-on-cpanel-as-a-pro

**Comment by ByAnthony** [comment]
Hi @Uri3lArchangel 👋 Have you managed with the solution provided by @etugbeh-plateaumed? I have followed it but getting a blank page and `Failed to load resource: the server responded with a status of 404 ()`

**Reply by ByAnthony** [reply]
Just in case that could be useful, this how I did it: [deploy on CPanel](https://github.com/ByAnthony/newzealandtunnellers/blob/main/docs/deployment.md)
